app/views/staff/speakers/index.html.haml

```diff
@@ -16,17 +16,26 @@
           %th
           %th
           %th
-          %th
         %tr
           %th Name
           %th Email
-          %th Proposal Title
-          %th Proposal Status
+          %th Program Session Title
+          %th Actions
       %tbody
-        - proposals.each do |proposal|
-          - proposal.speakers.each do |speaker|
-            %tr
-              %td=link_to speaker.name, edit_profile_event_staff_speaker_path(event, speaker)
-              %td= speaker.email
-              %td= link_to proposal.title, event_staff_proposal_path(event, proposal)
-              %td= proposal.state_label(small: true)
+        - @program_session_speakers.each do |speaker|
+          %tr{ id: "speaker-#{speaker.id}" }
+            %td= speaker.name
+            %td= speaker.email
+            %td= link_to speaker.program_session.title, "#" # should this go to a program session show page?
+            %td{ id: "speaker-action-buttons-#{speaker.id}" }
+              %span>
+                = link_to "Edit", edit_event_staff_speaker_path(current_event, speaker), class: "btn btn-primary btn-xs"
+              %span>
+                = link_to "Add Speaker", new_event_staff_program_session_speaker_path(current_event, speaker.program_session), class: "btn btn-primary btn-xs"
+              %span>
+                - if speaker.program_session.speakers.count > 1
```

Comment:
You'll want to move this logic into a method on either speaker or program session.  The view shouldn't know  this much about the internal logic/behavior of either object.